src/TypeSpec.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -75,20 +78,24 @@ import {
 import { capitalize, getFullNamespaceString, getTypeName } from "./utils.js";
 import { InputTypeKind } from "../type/inputTypeKind.js";
 import { InputIntrinsicTypeKind } from "../type/inputIntrinsicTypeKind.js";
-import { fromSdkEnumType } from "../type/converter.js";
+import { fromSdkEnumType, fromSdkModelType } from "../type/converter.js";
+import { $lib } from "../emitter.js";
```

Comment:
I expect we could delete a lot of lines in this file?

Reply:
In the latest codes, we do removed a lot of imports.